internal/provisioner/plain/pkg/git/git.go

```diff
@@ -0,0 +1,78 @@
+package git
+
+import (
+	"errors"
+	"fmt"
+	"net/url"
+	"strings"
+
+	"github.com/operator-framework/rukpak/api/v1alpha1"
+)
+
+const (
+	defaultDirectory = "./manifests"
+	defaultBranch    = "main"
```

Comment:
Is it possible to ask the git repo for it's default branch rather than hardcoding it?

Reply:
I looked and it was a bit non-trivial. Requiring to specify a branch makes sense IMO.